tasks/generateFeatureIndex.js

```diff
@@ -14,7 +14,7 @@ function run(callback) {
     let featureNames = [];
 
     files.forEach((filePath) => {
-      const filePathSplit = filePath.split(path.sep);
+      const filePathSplit = filePath.split('/');
```

Comment:
Yep. path.sep didn't work because it is / on Windows but the split has to be done on the \.